frontend/vue/components/Syllabus/SyllabusGeneralInformation.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="syllabus-general-information">
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">Instructor:</span> <slot name="instructor" />
+    </div>
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">Location:</span> <slot name="location" />
+    </div>
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">University:</span> <slot name="university" />
+    </div>
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">Office Hours:</span> <slot name="office-hours" />
+    </div>
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">Class Hours:</span> <slot name="class-hours" />
+    </div>
+    <div class="syllabus-general-information__item">
+      <span class="syllabus-general-information__item__heading">Email:</span> <slot name="email" />
```

Comment:
Question for @gracelindsell: 

Should we code the email in the syllabus view, to be a clickable link? or just plain-text is fine?

Reply:
(updating comment to a non-blocking comment... we can always address this later)